src/rust/ide/view/graph-editor/src/builtin/visualization/java_script/geoMap.js

```diff
@@ -206,6 +220,19 @@ class GeoMapVisualization extends Visualization {
     updateLayers() {
         this.deckgl.setProps({
             layers: [this.makeScatterLayer()],
+            getTooltip: ({ object }) =>
+                object && {
+                    html: `<div>${object.label}</div>`,
+                    style: {
+                        backgroundColor: this.labelBackgroundColor,
+                        fontSize: '12px',
+                        borderRadius: '14px',
+                        borderTopLeftRadius: '1px',
+                        fontFamily: 'DejaVuSansMonoBook, sans-serif',
```

Comment:
constants - font should be moved to section. The same applies to font size and every other literal, like corner radius